docker/Dockerfile

```diff
@@ -6,61 +6,40 @@ RUN dnf install -y epel-release.noarch\
     && rm -rf /var/cache/dnf
 
 # PKI stuff
-RUN dnf install -y https://repo.opensciencegrid.org/osg/3.6/osg-3.6-el9-release-latest.rpm\
-    && dnf clean all \
-    && rm -rf /var/cache/dnf
-
-RUN dnf install -y osg-pki-tools\
-    && dnf clean all \
-    && rm -rf /var/cache/dnf
-
-RUN dnf install -y voms\
-    && dnf clean all \
-    && rm -rf /var/cache/dnf
-
-RUN  dnf install -y voms-clients-cpp\
-    && dnf clean all \
-    && rm -rf /var/cache/dnf
-
-RUN mkdir -p /etc/grid-security \
-    && rm -rf /etc/grid-security/certificates \
-    && ln -s /cvmfs/grid.cern.ch/etc/grid-security/certificates /etc/grid-security/
-
+# xrootd client
+# Python and libs
+RUN dnf install -y https://repo.opensciencegrid.org/osg/3.6/osg-3.6-el9-release-latest.rpm && \
+    dnf install -y osg-pki-tools && \
```

Comment:
While it's nice to have the repo on it's own dnf line, I think you can go further by collapsing the rest of the package installs into one command.